snappi/snappicommon.py

```diff
@@ -143,13 +143,20 @@ def parent(self):
     def _get_property(self, name, default_value=None, parent=None, choice=None):
         if name not in self._properties or self._properties[name] is None:
             if isinstance(default_value, type) is True:
+                default_value._CHECK_REQUIRED = False
                 self._properties[name] = default_value(parent=parent, choice=choice)
             else:
                 self._properties[name] = default_value
         return self._properties[name]
 
     def _set_property(self, name, value, choice=None):
-        self._properties[name] = value
+        # if name in self._REQUIRED and value == None and self._CHECK_REQUIRED is True:
+        #     msg = "please pass the mandatory args {} ".format(self._REQUIRED)
+        #     raise Exception(msg + "%s is a mandatory property, needs to initialize" % name)
+        if name in self._DEFAULTS and value == None:
```

Comment:
replace all `value == None` with `value is None`

Reply:
addressed